File: onebot/src/main/java/cn/evole/onebot/sdk/action/ActionRaw.java
package cn.evole.onebot.sdk.action;


import com.google.gson.annotations.SerializedName;
import lombok.Data;

/**
 * Description:
 * Author: cnlimiter
 * Date: 2022/9/13 22:05
 * Version: 1.0
 */
@Data
public class ActionRaw{

    @SerializedName("status")
    public String status;
    @SerializedName("retcode")
    public int retCode;
    @SerializedName( "echo")
    public long echo;

}

File: onebot/src/main/kotlin/sdk/action/ActionRaw.kt
package cn.evolvefield.onebot.sdk.action

import com.google.gson.annotations.SerializedName
import lombok.Data

@Data
class ActionRaw {
    @SerializedName("status")
    var status = "failed"
    @SerializedName("retCode")
    var retCode = 1400
    @SerializedName("echo")
    var echo: String? = null
}
